OpenOversight/app/main/views.py

```diff
@@ -724,6 +725,9 @@ def add_department():
                 Job(job_title="Not Sure", order=0, department_id=department.id)
             )
             db.session.flush()
+            Department(id=department.id).remove_database_cache_entries(
```

Comment:
Why is this not `remove_database_cache_entries(None, [KEY_DEPTS_BY_STATE])`. Then you wouldn't have to change the `remove_database_cache_entries` function I think. And it is clearer that this will affect all departments

Reply:
`remove_database_cache_entries` is not a static function, so it requires some sort of instantiation. I can make the change, but it does require an extensive re-write.

Reply:
Sorry, I was referring to the `remove_database_cache_entries` function in the database_cache file, not the helper function that forwards `department.remove_database_cache_entries(x)` to  `remove_database_cache_entries(department, x)`. That one can remain as is (and should keep working) I believe and then I don't think much has to be changed